test/scripts/openshift-ci/deploy.cma.sh

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env bash
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu # Exit on error and undefined variables
+
+SUBSCRIPTION_NAME="openshift-custom-metrics-autoscaler-operator"
+KEDA_NAMESPACE="openshift-keda"
+KEDA_OPERATOR_POD_LABEL="app=keda-operator"
+KEDA_METRICS_API_SERVER_POD_LABEL="app=keda-metrics-apiserver"
+KEDA_WEBHOOK_POD_LABEL="app=keda-admission-webhooks"
+
+# Helper function to wait for a pod with a given label to be created
+waitforpodlabeled() {
```

Comment:
We already have those functions in both `deploy.ossm.sh` and `deploy.serverless.sh` for... reasons (?). Not sure if that's the right time to make a shared `pod.func.sh` which we could source in both? WDYT?